challenges/literal-types/user.ts

```diff
@@ -7,5 +7,6 @@ type LiteralFunction = unknown;
 const literalString = undefined;
 const literalTrue = undefined;
 const literalNumber = Math.random() > 0.5 ? undefined : undefined;
+const almostPi = undefined;
```

Comment:
I think, we don't need almostPi declared. TypeHero challenges expects the user read the tests and figure out what needs to be created. "[Type Aliases](https://typehero.dev/challenge/type-aliases)" challenge has a note regarding this as well.

> As you continue your journey with TypeScript (and TypeHero!) you should get into the habit of closely consulting tests as you go. The tests can help you ensure that your implementation is correct, and quite often that means filling in missing pieces in your implementation.

Reply:
shit you’re right, can someone undo that

Reply:
Ah yes, well seen. I've just reverted it ;)